acceptance/gossip_peerings_test.go

```diff
@@ -141,6 +141,8 @@ func TestGossipPeerings(t *testing.T) {
 // re-bootstrapped after a time when all nodes were down
 // simultaneously.
 func TestGossipRestart(t *testing.T) {
+	t.Skipf("TODO(pmattis): #4445")
```

Comment:
It failed for me on CI 15 hours ago: https://circleci.com/gh/petermattis/cockroach/321?utm_campaign=build-failed&utm_medium=email&utm_source=notification